src/client_data.rs

```diff
@@ -0,0 +1,110 @@
+use fastly::error::anyhow;
```

Comment:
```suggestion
//! Compute@Edge starter kit for Network Error Logging
use fastly::error::anyhow;
```

Reply:
I know this duplicates info in `fastly.toml` etc., but a `//!` will show up in the rustdoc output.

Reply:
I made a silly mistake, didn't realize that this was `client_data.rs` and not `main.rs`.

You could trim or edit these module level comments if you'd like, but I don't think it's especially important. I'm happy to recycle the :heavy_check_mark: if you elect to change them.